### PR TITLE
[FIX] - Adding references to tutorials

### PR DESCRIPTION
**Pull request by nhussein11**
This PR aims to add a reference to tutorials at the bottom of the pages that can be linked with a tutorial

### CODE REVIEW

File: develop/parachains/get-started/build-custom-parachains.md [hunk 1]
```diff
@@ -57,3 +57,25 @@ Each of these steps is designed to build on the last, helping ensure that your c
...
+
+    Learn to build and manage Polkadot SDK-based local solochains. Covers setup, networking, authorization, monitoring, and upgrades.
+
+    [:octicons-arrow-right-24: Reference](tutorials/polkadot-sdk/parachains/local-chain/)
+
+-   __Connect to a Relay Chain__
+
+    ---
+
+    Learn how to set up a local relay chain for parachain testing including managing gensis state, TestNet deployment, and managing collator nodes.
```
Comment (nhussein11) [inline develop/parachains/get-started/build-custom-parachains.md:77]:
```suggestion
    Learn how to set up a local relay chain for parachain testing, including managing the genesis state, TestNet deployment, and managing the collator nodes.
```

**Review by 0xLucca — APPROVED** [review]
LGTM!

**Review by eshaben — COMMENTED** [review]
Just wanted to start a quick discussion - why not keep the sections as "Where to Go Next"? A lot of other pages still use it

**Comment by dawnkelly09** [comment]
> Just wanted to start a quick discussion - why not keep the sections as "Where to Go Next"? A lot of other pages still use it

That came from a Slack convo with Alberto. Dropped you a link in DM.

File: develop/parachains/get-started/build-custom-parachains.md [hunk 2]
```diff
@@ -52,8 +52,9 @@ Building a custom blockchain with the Polkadot SDK involves several core steps,
 
 Each of these steps is designed to build on the last, helping ensure that your custom blockchain is functional, optimized, and ready for deployment within the Polkadot ecosystem or beyond.
 
+<!-- This page will be removed  -->
```
Comment (nhussein11) [inline develop/parachains/get-started/build-custom-parachains.md:55]: This page will be removed in #246

**Review by eshaben — APPROVED** [review]
LGTM!

**Pull request merged**